index.html

```diff
@@ -15,7 +15,12 @@
     <header>
       <nav>
         <div class="left-side"><a class="logo" href="">My Logo</a></div>
-        <div class="right-side"><img src="images/Union.svg" alt="union"></div>
+        <div class="right-side">
+          <a class="desktop-nav" href="http://">Portfolio</a>
+          <a class="desktop-nav" href="http://">About</a>
+          <a class="desktop-nav"  href="http://">Contact</a>
+          <img src="images/Union.svg" alt="union">
+        </div>
       </nav>
     </header>
```

Comment:
- [ Optional ] Kindly fix the unnecessary space above on the Top Navigation of your portfolio.

![Capture](https://user-images.githubusercontent.com/93598189/199968388-7195fa6c-78de-4d30-8d4d-bf7a0c480251.PNG)